mlem/core/metadata.py

```diff
@@ -70,22 +65,18 @@ def save(
         fs: FileSystem for the `path` argument
         index: Whether to add object to mlem project index
         external: if obj is saved to project, whether to put it outside of .mlem dir
-        description: description for object
         params: arbitrary params for object
-        labels: labels for object
-        update: whether to keep old description/labels/params if new values were not provided
+        update: whether to keep old params if new values were not provided
 
     Returns:
         None
     """
-    if update and (description is None or params is None or labels is None):
+    if update and params is None:
```

Comment:
i'd say let's overwrite params each time

Reply:
Then we don't need an `update` flag at all, right?

Reply:
yes